src/client/app/components/groups/EditGroupComponent.jsx

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This change is straight up wrong. Please undo all your changes to EditGroupComponent.